force_bdss/core/verifier.py

```diff
@@ -178,16 +182,18 @@ def _check_data_source(data_source_model, layer_number):
             global_error="Missing output slot name assignment "
                          "in layer {}".format(layer_number)))
 
+    #: Check if any datasources have all outputs unnamed
     row_index_errors = []
```

Comment:
I would change this logic to check only if all are undefined. Or as we said, the possibility that all could be undefined is not zero, although unlikely.